shared_crates/ui/src/scroll_area.rs

```diff
@@ -68,3 +72,111 @@ pub fn ScrollArea(
         }
     }
 }
+
+
+/// A scroll box view that can be used to scroll its child with a scroll bar.
+#[element_component]
+pub fn ScrollBoxView(
+    hooks: &mut Hooks,
+    /// Width of the scroll area
+    min_width: f32,
+    /// Height of the scroll area
+    min_height: f32,
+    /// Distance of scrolling in y axis
+    scroll_height: f32,
```

Comment:
Scroll height should be automatically measured internally. Use the `MeasureSize` component on the inner content. You can also see how it's done in the ScrollArea component

Reply:
The `MesureSize` does not return a proper height of a `Flow`, is this something that should be fixed first?

Reply:
Hm that sounds fishy, it's a very simple component. Check the source of it